e2e/components/Heading.test.ts

```diff
@@ -1,33 +1,74 @@
 import {test, expect} from '@playwright/test'
 import {visit} from '../test-helpers/storybook'
-import {themes} from '../test-helpers/themes'
 
 test.describe('Heading', () => {
   test.describe('Default', () => {
-    for (const theme of themes) {
-      test.describe(theme, () => {
-        test('default @vrt', async ({page}) => {
-          await visit(page, {
-            id: 'components-heading--default',
-            globals: {
-              colorScheme: theme,
-            },
-          })
-
-          // Default state
-          expect(await page.screenshot()).toMatchSnapshot(`Heading.Default.${theme}.png`)
-        })
-
-        test('axe @aat', async ({page}) => {
-          await visit(page, {
-            id: 'components-heading--default',
-            globals: {
-              colorScheme: theme,
-            },
-          })
-          await expect(page).toHaveNoViolations()
-        })
-      })
-    }
+    test('default @vrt', async ({page}) => {
+      await visit(page, {
+        id: 'components-heading--default',
+      })
+
+      // Default state
+      expect(await page.screenshot()).toMatchSnapshot(`Heading.Default.png`)
```

Comment:
aww? we don't tests for all themes?

Reply:
Its just testing for font-size so color doesn't matter 😄